examples/key_based_replication/connector.py

```diff
@@ -0,0 +1,154 @@
+# This is a simple example for how to work with the fivetran_connector_sdk module.
+# This shows key based replication from DB sources.
```

Comment:
Add more details on what is and how key based replication works. It's better to add README file in this folder.

Reply:
Added in comments and not in readMe